server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/EmailSubmissionSet.scala

```diff
@@ -87,4 +87,5 @@ object EmailSubmissionCreationRequest {
 }
 
 case class EmailSubmissionCreationRequest(emailId: MessageId,
+                                          identityId: Option[Id],
```

Comment:
no associated test?

Reply:
That's contributed on my personal time. I did not took the time to do it.

Reply:
Very good catch BTW